INN/umbrella-citylimits
# Add honeypot for newsletter signup forms

## Changes

This pull request makes the following changes:

- Adds a fake hidden `name` field to the newsletter signup forms to trick bots into filling it out so we know not to actually submit the form

## Why



For #163

## Testing/Questions

Features that this PR affects:

- Newsletter subscription forms


Questions that need to be answered before merging:

- [ ] Does this cover all forms, or are some missing?

Steps to test this PR:

1. Fill out a newsletter form as usual and make sure it submits successfully
2. Reload, fill out a newsletter form as usual and unhide the hidden field and give it a value. Submit the form and make sure instead of a real submit success message, you just see `Submitted.`
